{com_pos_x: -0.0002178162356267396, com_pos_y: -5.379150827036468e-05, com_pos_z: 0.0022167267565283725,
  force_zero_x: -0.34599856744691337, force_zero_y: -2.7198373216263061, force_zero_z: -14.671800440173246,
  gravity_frame: /optitrak, mass: 0.062340672111074517, wrench_frame: /tool_netft_raw_frame,
  wrench_topic: /tool_netft/wrench_raw, torque_zero_x: -0.074936210242300613, torque_zero_y: 0.046850263249184024,
  torque_zero_z: 0.088066462569410919, wrench_base_frame: /optitrak, wrench_location_frame: /scratcher}
